content/source/docs/cloud/api/oauth-clients.html.md

```diff
@@ -296,8 +299,10 @@ Key path                             | Type   | Default | Description
 -------------------------------------|--------|---------|------------
 `data.type`                          | string |         | Must be `"oauth-clients"`.
 `data.attributes.name`               | string | (previous value)  | An optional display name for the OAuth Client. If set to `null`, the UI will default to the display name of the VCS provider.
-`data.attributes.key`                | string | (previous value) | The OAuth client key.
-`data.attributes.secret`             | string | (previous value) | The OAuth client secret.
+`data.attributes.key`                | string | (previous value) | The OAuth Client key. It sometimes refers to a Consumer Key or Application Key or some other client key for the VCS provider.
```

Comment:
```suggestion
`data.attributes.key`                | string | (previous value) | The OAuth Client key. It can refer to a Consumer Key, Application Key, or another type of client key for the VCS provider.
```